.github/workflows/codeql-analysis.yml

```diff
@@ -0,0 +1,67 @@
+# For most projects, this workflow file will not need changing; you simply need
+# to commit it to your repository.
+#
+# You may wish to alter this file to override the set of languages analyzed,
+# or to provide custom queries or build logic.
+#
+# ******** NOTE ********
+# We have attempted to detect the languages in your repository. Please check
+# the `language` matrix defined below to confirm you have the correct set of
+# supported CodeQL languages.
+#
+name: "CodeQL"
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    # The branches below must be a subset of the branches above
+    branches: [ main ]
+  schedule:
```

Comment:
@jshcodes checkout the action dry run. Shows as green but throws an error the base branch isn't found. It's appropriately set to ``main`` here though.

Reply:
Ref https://github.com/CrowdStrike/falconpy/pull/93/checks?check_run_id=2198470762

Reply:
Think this is related to this being the first check in for it, so the initial comparison set doesn't exist.

[Reasons for the missing analysis message](https://docs.github.com/en/code-security/secure-coding/setting-up-code-scanning-for-a-repository#reasons-for-the-missing-analysis-message)